spec/support/schemas_camelized/va_forms/forms.json

```
@@ -54,10 +54,7 @@
               },
               "sha256": {
                 "type": "string"
-              },
-              "lastSha256Change": {
-                "type": ["string", "null"]
-              },
+              }
```

```suggestion
              },
```

This json is invalid.

---

FIxed. Yeah, this one is tricky. I can't reproduce it locally. But I found some records in staging that are blowing up, so I'm going to copy them down locally and try to troubleshoot